scripts/test-alamofire.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
`m`: It would be great to keep the explanation of 
https://github.com/getsentry/sentry-cocoa/blob/89b12ebb5452bf2787046df229892098a87e33d6/.github/workflows/integration-tests.yml#L16-L22

We could add it as a comment here.